projects/core/src/lib/map/map-layer.component.ts

```diff
@@ -311,12 +320,16 @@ export class OlRasterLayerComponent
 
         this.source.setTileLoadFunction((tile, src) => {
             const client = new XMLHttpRequest();
-
-            const sub = this.projectService.createQueryAbortStream().subscribe(() => {
-                (tile as any).getImage().src = 'assets/images/empty_tile.png';
-                client.abort();
-                sub.unsubscribe();
-                this.source.dispatchEvent('tileloaderror');
+            const tileZoomLevel: number = (tile as any).getTileCoord()[0];
+            const tileResolution = this.mapService.getView().getResolutionForZoom(tileZoomLevel);
```

Comment:
I guess you could use `getTileCoord` instead: https://openlayers.org/en/latest/apidoc/module-ol_Tile-Tile.html

It gives you x, y, and zoom level z.

Reply:
I already use it